taichi/ir/statements.h

```diff
@@ -263,9 +263,15 @@ class ExternalPtrStmt : public Stmt {
  public:
   LaneAttribute<Stmt *> base_ptrs;
   std::vector<Stmt *> indices;
+  std::vector<int> element_shape;
```

Comment:
FYI ir_printer.cpp also need to be updated to include this info for debugging purpose. ;)

Reply:
Thanks! I'm looking into this!

Reply:
I've encountered some problems here. Print the element shape in IR would crash some place in module.py. Is there any  document for IR print conventions?

Reply:
Done :)